Monika After Story/game/script-topics.rpy

```diff
@@ -15151,6 +15151,46 @@ label monika_songwriting:
     m 1eua "I'd love to sing with you [_if] I come to your world, [player]."
     return
 
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_physicalmedia",
+            category=['media'],
+            prompt="Physical media",
+            action=EV_ACT_RANDOM
```

Comment:
I can't see a `conditional` here.
`action`, `conditional`, these two things must be used at the same time.

If you want to make this topic an unconditional random topic, the best way is to delete `action` and `conditional`, and then simply write `random=True`.